istioctl/pkg/verifier/verifier.go

```diff
@@ -106,6 +115,26 @@ func (v *StatusVerifier) verifyInstallIOPRevision() error {
 	return v.reportStatus(crdCount, istioDeploymentCount, err)
 }
 
+func getRevision(istioNamespace, kubeConfig, ctx string) (string, error) {
+	var revision string
+	cfg, err := kube.BuildClientConfig(kubeConfig, ctx)
+	if err != nil {
+		return "", err
+	}
+	kubeClient, err := kube.NewExtendedClient(kube.NewClientConfigForRestConfig(cfg), "")
+	if err != nil {
+		return "", fmt.Errorf("failed to connect Kubernetes API server, error: %v", err)
+	}
+	pods, err := kubeClient.PodsForSelector(context.TODO(), istioNamespace, "app=istiod")
+	if err != nil {
+		return "", fmt.Errorf("failed to fetch istiod pod, error: %v", err)
+	}
+	for _, pod := range pods.Items {
+		revision = pod.ObjectMeta.GetLabels()[label.IstioRev]
```

Comment:
If there are multiple `istiod` instances in `istioNamespace` this will pick an arbitrary one and use its revision? Is that the behavior we want?

Reply:
Yes, it picks random revision and runs `verify-install` on it. If the user knows and wants to run `verify-install` on the specific control plane they can use `istioctl verify-install --revision 1-8-0`.